explanations/python-imports-check-typo.md

```diff
@@ -0,0 +1,16 @@
+Not all python packages include tests. However it should be possible, at a minimum, to "smoke test" every python package my simply checking that the modules it declares can be `import`ed.
```

Comment:
```suggestion
Not all python packages include tests. However it should be possible, at a minimum, to "smoke test" every python package by simply checking that the modules it declares can be `import`ed.
```

This should probably added to all python packages regardless of tests.

Reply:
Right now, the logic in the check only requires the package to have either a checkPhase or a pythonImportsCheck. Do you think it should be changed to require a pythonImportsCheck unconditionally?

Reply:
I am not completely sure but it does not hurt.